sqlserver/datadog_checks/sqlserver/sqlserver.py

```diff
@@ -259,6 +275,21 @@ def _make_metric_list_to_collect(self, custom_metrics):
                 cfg = {'name': name, 'table': table, 'column': column, 'tags': tags}
                 metrics_to_collect.append(self.typed_metric(cfg_inst=cfg, table=table, column=column))
 
+        # Load DB Fragmentation metrics
+        if is_affirmative(self.instance.get('include_db_fragmentation_metrics', False)):
+            db_name = self.instance.get('database', self.connection.DEFAULT_DATABASE)
+            object_name = self.instance.get('object_name', None)
```

Comment:
```suggestion
            object_name = self.instance.get('object_name')
```
That's the default behaviour of `get`.